packages/geoprocessing/src/aws/helpers.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  InvokeCommand,
+  LambdaClient,
+  InvocationResponse,
+} from "@aws-sdk/client-lambda";
+import geobuf from "geobuf";
+import Pbf from "pbf";
+import { Sketch, SketchCollection } from "../types/sketch.js";
+import {
+  Polygon,
+  MultiPolygon,
+  GeoprocessingRequestModel,
+  GeoprocessingRequestParams,
+  JSONValue,
+} from "../types/index.js";
+import { genTaskCacheKey } from "../helpers/genTaskCacheKey.js";
+import { GeoprocessingTask, GeoprocessingTaskStatus } from "./tasks.js";
+import { byteSize } from "../util/byteSize.js";
+
+/**
+ * Runs a function on a specified lambda worker
+ * @param sketch
+ * @param projectName name of project in package.json
+ * @param functionName name of lambda worker function
```

Comment:
Consider changing this to "name of sync geoprocessing function to run as worker"

Reply:
Updated!